docs/site/sidebars/lb4_sidebar.yml

```
@@ -244,7 +244,7 @@ children:
         output: 'web, pdf'
 
       - title: 'Exposing GraphQL APIs'
```

Change the title in the sidebar to match the title of the actual page

```suggestion
      - title: 'Using OpenAPI-to-GraphQL'
```